.teamcity/Lib/Project.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed to the Ed-Fi Alliance under one or more agreements.
+ * The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ * See the LICENSE and NOTICES files in the project root for more information.
+ */
+package lib
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+
+object SharedLibrariesProject : Project({
+    name = "Shared Libraries"
+    id = RelativeId("SharedLibraries")
+    description = "Build configurations for shared libraries"
+
+    params {
+        param("api.semantic.version", "4.0.0")
```

Comment:
can we set this up to pull from a property file instead? thoughts?

Reply:
I'm not sure what you mean or how this would work. At least now the version is in source control, not configured via the UI and locked there.